packages/bot/src/components/quick-strategy.jsx

```diff
@@ -16,14 +16,13 @@ import {
 }                           from 'formik';
 import { localize }         from 'deriv-translations';
 import IconTradeType        from './icon-trade-types.jsx';
-import { InfoOutlineIcon }  from './Icons.jsx';
 import { connect }          from '../stores/connect';
 import config               from '../constants/index';
 import '../assets/sass/quick-strategy.scss';
 
 const MarketOption = ({ symbol }) => (
     <div key={symbol.value} className='quick-strategy__option'>
-        <UnderlyingIcon market={symbol.value} />
+        <Icon icon={`IcUnderlying${symbol.value}`} width={32} height={24} />
```

Comment:
Same here, dimensions set here may be exceeded.

```suggestion
        <Icon icon={`IcUnderlying${symbol.value}`} size={32} />
```